pyproject.toml

```diff
@@ -28,7 +28,7 @@ authors = [
 ]
 license = { file = "LICENSE" }
 dependencies = [
-    "nomad-lab>=1.3.4dev",
+    "nomad-lab>=1.3.6",
```

Comment:
Given that now we require `index-url` for the pip install, we can still keep using the dev version (>=1.3.4dev)

Reply:
But 1.3.4.dev fails, no? We should require what is actually required to run the plugin or what do you mean?

Reply:
No, 1.3.4dev works because it pulls whatever latest dev (1.3.7devNN) is available on the registry

Reply:
For the issue that Fabian is facing, I think using the index-URL should solve it for him as you mentioned on the issue, as now when pip installing, the latest dev of nomad-lab will be pulled. 

For the general milestone issue that we need a stable version as a dependency, we only have 1.3.6 as the current viable option (as 1.3.4 would fail the pytest). However, as you pointed out in today's discussion, there are some important issues not addressed in 1.3.6 and we should wait for a 1.3.7 https://github.com/FAIRmat-NFDI/nomad-material-processing/pull/79#issuecomment-2348368125

Reply:
That is not how a dependency works @ka-sarthak. If I write `numpy >= 1.3.4` in the toml it will pull the latest from PyPI which is `2.1.1` but by listing `numpy >= 1.3.4` in the toml we are saying that it should work with anything above and including that version! If we put 1.3.4.dev in our toml and someone has 1.3.4.dev32 installed locally they will not get nomad-lab updated and the solution class will not work. DO you understand what I mean?

Reply:
I think you are right when it comes to  `pip` updating the version of an already installed package. I always tested it with a fresh environment, so 1.3.4dev picked the latest dev, for example, 1.3.7.dev{something}, and never encountered the issue you pointed above. Let's use 1.3.6 then.